### DISCUSSION TITLE
[Question] Store/restore design state

### DISCUSSION THREAD

**Discussion by santoshgistto** [hm21/pro_image_editor#87] [Q&A] [answered]
Thanks for this package 

**Question**
is Store/restore design state supports ?

**Comment by hm21** [comment, marked as answer]
Hi,

I'm not sure if I understand you correctly. Did you mean to restore the state history? If yes, that's possible. You can read more about it in [readme](https://github.com/hm21/pro_image_editor?tab=readme-ov-file#import-export-state-history) which provides examples of how to implement it. 

This feature was requested within this [thread](https://github.com/hm21/pro_image_editor/issues/4#issue-2128129853).

**Comment by santoshgistto** [comment]
Thanks i was looking for this
